proposals/kubeflow-steering-committee-election-proposal.md

```diff
@@ -0,0 +1,125 @@
+# KSC Election Process Proposal
```

Comment:
Looking at steering committee charters from:
- [k8s](https://github.com/kubernetes/steering/blob/main/charter.md#direct-responsibilities-of-the-steering-committee): Define and evolve project and group[3](https://github.com/kubernetes/steering/blob/main/charter.md#footnote3) governance structures and policies[4](https://github.com/kubernetes/steering/blob/main/charter.md#footnote4).
- [kNative](https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#charter): Define and evolve project governance structures and policies, including project roles and how collaborators become members, approvers, leads, and/or administrators. This includes policy for the creation and administration of [working groups](https://github.com/knative/community/blob/main/working-groups/WORKING-GROUPS.md) and committees.
- [Istio](https://github.com/istio/community/blob/master/steering/CHARTER.md#istio-steering-committee):Establishing rules of governance for the Istio project, including creation and ratification of bylaws for the Steering Committee and Technical Oversight Committee.

In these examples, elections rules, steering committee charters, and other governance policies are purview of the steering committee. Any changes will go through public review & feedback, but the steering committee has the authority to make the changes. Appeals from public community members can be made to CNCF. I propose we adopt the same policy.

@theadactyl @jbottum

Reply:
In reality, I am only trying to prevent changes to the following things without community consensus:

1. The process by which the KSC is elected
2. The powers delegated to the KSC

So perhaps we can specifically mark this election document and the future "KSC Powers" document to require amending by the process I described.

I think it's fine to allow the KSC to manage its own "house rules" so to speak.

Reply:
Thinking about it more, there seems to be two options for how to achieve this "protection":

1. OPTION 1:
      - Proposals to change the "KSC Charter" and "KSC Election Process" can be raised by anyone, and adopted by the KSC, but only on a super-majority vote (with some protection for situations where there are a lot of vacancies)
1. OPTION 2:
      - Proposals to change the "KSC Charter" and "KSC Election Process" can be raised by anyone, and are adopted by a community vote.

Reply:
I understand the intention of the new proposal, but I think it results in over-governance. The KSC should have the autonomy to be self-governing and make policy changes. I am interested in hearing from others as well.

Reply:
Agreed, I think the risk of abuse here is quite low, and it's more important (IMO) to focus on support Steering Committee members being able to do their jobs effectively and efficiently.